README.md

```diff
@@ -51,3 +52,39 @@ We learned a lot from the following projects when building TVM.
   originates from Halide. We also learned and adapted some part of lowering pipeline from Halide.
 - [Loopy](https://github.com/inducer/loopy): use of integer set analysis and its loop transformation primitives.
 - [Theano](https://github.com/Theano/Theano): the design inspiration of symbolic scan operator for recurrence.
+
+
+# BYOC with DNNL
+
+## Requirement
+Tested with byoc: mxnet==1.8.0.post0  
+Tested with onednn: mxnet==2.0.0a0  
+
+## Build
+mkdir build  
+cp cmake/config.cmake build  
+cd build  
+  
+set(USE_LLVM ON)  
+set(USE_MKLDNN ON)  
+set(USE_DNNL_CODEGEN ON)  
+set(USE_OPENEM gnu)  
+  
+export CXXFLAGS="-I$HOME/built_onednn/include -I$HOME/oneDNN/build/include"  
+cmake .. -DEXTERN_LIBRARY_DNNL=$HOME/built_onednn/lib64/libdnnl.so -DMKLDNN_LIBRARY=$HOME/built_onednn/lib64/libmkldnn.so  
```

Comment:
Do we really need both of these two lib files (libdnnl.so and libmkldnn.so, i think they're same file with softlink).

Reply:
It truly needs the two lib files, or error occurs.

Reply:
I was referring to use the same lib name here:)